src/tests/mode.spec.ts

```diff
@@ -142,7 +142,8 @@ it('tracks changes to system preferences', async () => {
 it('stops tracking changes to system preferences when user sets a mode', async () => {
 	const { container, getByTestId } = render(Mode);
 	const rootEl = container.parentElement;
-	const dark = getByTestId('dark');
```

Comment:
the changes to this test file are not related to the tests breaking - even the first test is not working with this implementation. what I changed below was just to reset the state back to "default (dark mode)" so that the order of the tests is not important. right now, they need to be ran in a specific order which is not nice I think.

the changes below in this file works fine with the existing implementation :)